internal/policy/verify.go

```diff
@@ -129,6 +140,93 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 	return nil
 }
 
+// VerifyCommit verifies the signature on the specified commits (identified by
+// their hash or via a reference that is resolved). For each commit, the policy
+// applicable when the commit was first recorded (directly or indirectly) in the
+// RSL is used.
+func VerifyCommit(ctx context.Context, repo *git.Repository, ids []string) map[string]string {
```

Comment:
```suggestion
func VerifyCommit(ctx context.Context, repo *git.Repository, ids ...string) map[string]string {
```

Might be a good place for a variadic so you don't always have to pass in an array if you only want to verify a single commit.